docs/promises/create-account.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 3
+---
+
+# Creating Accounts
+
+You might want to create an account from a contract for many reasons. One example:
+You want to [progressively onboard](https://www.youtube.com/watch?v=7mO4yN1zjbs&t=2s) users, hiding the whole concept of NEAR from them at the beginning, and automatically create accounts for them (these could be child accounts of your main contract, such as `user123.some-cool-game.near`).
+
+Since an account with no balance is almost unusable, you probably want to combine this with the token transfer from [the last page](./token-tk.md). Here's how to do it:
+
+```rust
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::{json_types::U128, near_bindgen, Balance, AccountId, Promise};
+
+const INITIAL_BALANCE: Balance = 250_000_000_000_000_000_000_000; // 2.5e23yN, 0.25N
+
+#[near_bindgen]
+#[derive(Default, BorshDeserialize, BorshSerialize)]
```

Comment:
@austinabell, if there is an empty struct can we just skip the deserailazing/serializing step and so we won't even need of the `Borsh*` macros? or even the `#[near_bindgen]`